fetch.bs

```diff
@@ -2697,55 +2701,60 @@ the request.
      <tbody><tr>
       <th><a for=header>Name</a>
       <th><a for=header>Value</a>
-     <tr>
-      <td>`<code>DPR</code>`
-      <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#dpr>dpr value</a>
      <tr>
       <td>`<code>Save-Data</code>`
       <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#save-data>save-data value</a>
-     <tr>
-      <td>`<code>Viewport-Width</code>`
-      <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#viewport-width>viewport-width value</a>
     </table>
 
-   <li>
-    <p>If <var>request</var> is a <a>subresource request</a>, then:
-
-    <ol>
-     <li>
-      <p><a for=list>For each</a> <var>hintName</var> of <var>request</var>'s
-      <a for=client>client hints list</a>:
-
-      <ol>
-       <li>
-        <p>Let <var>value</var> be the first matching statement, switching on <var>hintName</var>:
-
-        <dl class=switch>
-         <dt>`<code>DPR</code>`
-         <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#dpr>dpr value</a>
-         <dt>`<code>Save-Data</code>`
-         <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#save-data>save-data value</a>
-         <dt>`<code>Viewport-Width</code>`
-         <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#viewport-width>viewport-width value</a>
-         <dt>`<code>Width</code>`
-         <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#width>width value</a>
-        </dl>
-
-       <li><p>A user agent should <a for="header list">append</a>
-       <var>hintName</var>/<var>value</var> to <var>request</var>'s <a for=request>header list</a>.
-      </ol>
-
-     <li><p>Let <var>record</var> be a new
-     <a for="fetch group">fetch record</a> consisting of
-     <var>request</var> and this instance of the
-     <a for=/>fetch</a> algorithm.
-
-     <li><p>Append <var>record</var> to <var>request</var>'s
-     <a for=request>client</a>'s
-     <a for=fetch>fetch group</a> list of
-     <a for="fetch group">fetch records</a>.
-    </ol>
-  </ol>
+    <li>
+
+     <p class=note>The following step should be applicable only for same-origin requests. See
+     <a href="https://github.com/whatwg/fetch/issues/800">issue</a> for more details.</p>
```

Comment:
This should be class=XXX and should be placed at the end of the step per convention.